pkg/controller/acicontainersoperator.go

```diff
@@ -353,48 +408,57 @@ func (t *OperatorHandler) ObjectCreated(obj interface{}) {
 
 	log.Info("Platform flavor is ",acicontainersoperator.Spec.Flavor)
 
-	//if (acicontainersoperator.Spec.Flavor == "openshift-4.3") {
-	//
-	//	log.Info("Updating Network Status of the Openshift Object....")
-	//	cfg, err := config.GetConfig()
-	//	scheme := runtime.NewScheme()
-	//	err = networkv1.Install(scheme)
-	//	if err != nil {
-	//		log.Error(err)
-	//		return
-	//	}
-	//
-	//	rclient, err := client.New(cfg, client.Options{Scheme: scheme})
-	//	if err != nil {
-	//		return
-	//	}
-	//
-	//	clusterConfig := &configv1.Network{
-	//		TypeMeta:   metav1.TypeMeta{APIVersion: configv1.GroupVersion.String(), Kind: "Network"},
-	//		ObjectMeta: metav1.ObjectMeta{Name: "cluster"},
-	//	}
-	//
-	//	err = rclient.Get(context.TODO(), types.NamespacedName{
-	//		Name: "cluster",
-	//	}, clusterConfig)
-	//	if err != nil {
-	//		log.Info(err)
-	//	}
-	//	log.Info("Config Spec is  ", clusterConfig.Spec)
-	//	clusterConfig.Status.ClusterNetwork = clusterConfig.Spec.ClusterNetwork
-	//	clusterConfig.Status.NetworkType = clusterConfig.Spec.NetworkType
-	//	clusterConfig.Status.ServiceNetwork = clusterConfig.Spec.ServiceNetwork
-	//
-	//	log.Info("Status is ", clusterConfig.Status)
-	//
-	//	ctx := context.TODO()
-	//	err = rclient.Update(ctx, clusterConfig)
-	//	if err != nil {
-	//		log.Info(err)
-	//		return
-	//	}
-	//}
+	if (acicontainersoperator.Spec.Flavor >= "openshift-4.3") {
 
+		clusterConfig := &configv1.Network{
+			TypeMeta:   metav1.TypeMeta{APIVersion: configv1.GroupVersion.String(), Kind: "Network"},
+			ObjectMeta: metav1.ObjectMeta{Name: "cluster"},
+		}
+
+		cfg, err := config.GetConfig()
+		scheme := runtime.NewScheme()
+		err = configv1.Install(scheme)
+		if err != nil {
+			log.Error(err)
+			return
+		}
+
+		rclient, err := client.New(cfg, client.Options{Scheme: scheme})
+		if err != nil {
+			return
+		}
+
+		err = rclient.Get(context.TODO(), types.NamespacedName{
+			Name: "cluster",
+		}, clusterConfig)
+		if err != nil {
+			log.Info(err)
+		}
+
+		log.Info("Config Spec is  ", clusterConfig.Spec)
+
+		status  := &configv1.NetworkStatus{}
+
+
+		log.Info("Current clusterConfig.Status is ", clusterConfig.Status)
+
+		if !reflect.DeepEqual(clusterConfig.Status,status){
+			log.Info("Updating Network Status of the network Object....")
```

Comment:
Can you be more specific (i.e. name of the resource) about the Netowork Object that is being updated here?

Also, why does "Status" start with an upper case? In general, can we exercise a little more discipline when writing log messages at least at the Info level?